crates/oxc_language_server/src/main.rs

```diff
@@ -54,11 +56,17 @@ struct Options {
     run: Run,
     enable: bool,
     config_path: String,
+    use_nested_configs: bool,
```

Comment:
Allow opting into nested configs. If false, there shouldn't be any behavior change.